scipy/stats/_multivariate.py

```diff
@@ -507,6 +512,110 @@ def pdf(self, x, mean=None, cov=1, allow_singular=False):
         out = np.exp(self._logpdf(x, mean, psd.U, psd.log_pdet, psd.rank))
         return _squeeze_output(out)
 
+    def _cdf(self, x, mean, cov, maxpts, abseps, releps):
+        """
+        Parameters
+        ----------
+        x : ndarray
+            Points at which to evaluate the cumulative distribution function.
+        mean : ndarray
+            Mean of the distribution
+        cov : array_like
+            Covariance matrix of the distribution
+        maxpts: integer
+            The maximum number of points to use for integration
+        abseps: float
+            Absolute error tolerance
+        releps: float
+            Relative error tolerance
+
+        Notes
+        -----
+        As this function does no argument checking, it should not be
+        called directly; use 'cdf' instead.
+
+        .. versionadded:: 0.19.0
+
+        """
+        lower = np.full(mean.shape, -np.inf)
+        # mvnun expects 1-d arguments, so process points sequentially
+        func1d = lambda x_slice: mvn.mvnun(lower, x_slice, mean, cov,
+                                           maxpts, abseps, releps)[0]
+        out = np.apply_along_axis(func1d, -1, x)
+        return _squeeze_output(out)
+
+    def logcdf(self, x, mean=None, cov=1, allow_singular=False, maxpts=100000,
+               abseps=1e-6, releps=1e-6):
```

Comment:
Compared KDE, this keeps the defaults for tolerances but changes the default maxpts, which is `1000*d` https://github.com/scipy/scipy/blob/v0.19.0/scipy/stats/mvn.pyf#L13 
Any reason for this specific value of 100000?